### ISSUE TITLE
pillow: Cannot access member "getbands" for type "Image"

### ISSUE THREAD

**Issue by tekumara** [microsoft/pylance-release#556]
## Environment data

-   Language Server version: Pylance language server 2020.10.3 (pyright b881f28d) starting
-   OS and version: macOS
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.7.9

## Expected behaviour

No type error, see [Image.getbands() docs](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.getbands) 

## Actual behaviour

```
Cannot access member "getbands" for type "Image"
  Member "getbands" is unknown
```

## Code Snippet / Additional information

```python
from typing import Tuple
import PIL.Image


def bands(pimage: PIL.Image.Image) -> Tuple[str]:
    return pimage.getbands()

```

**Comment by falkben** [comment]
For me I get

> Import "PIL" could not be resolved from source"

on just the simple `from PIL import Image`

**Comment by jakebailey** [comment]
I believe that's a different issue; that happens when pillow isn't found in the environment at all, i.e. it's not installed in the selected interpreter.

The stub will always be present, as it's bundled with Pylance (so we'll "resolve" it, but warn that the code may not run).

I haven't had the time to go back and fix up these stubs, unfortunately, but I don't think that impacts resolving the modules themselves.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202150-5-may-2021
